README.md

```diff
@@ -60,6 +60,17 @@ If your version of NativeScript is incompatible with the extension you will see
     npm run launch-as-server # launches the debug adapter in server mode
     # execute this in a separate terminal
     npm run test-mac # run tests on ios device
-    ```    
+    ```
 
 4. To install the extension drag and drop the `nativescript-*.*.*.vsix` package in the VS Code.
+
+### How to disable the analytics
+NativeScript Extension for Visual Studio Code collects usage data and sends it to Progress to help improve our products and services.
```

Comment:
Maybe we should add that it is not personally identifiable?

Is this synced with Garo?